parlai/crowdsourcing/tasks/turn_annotations_static/webapp/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Never heard of a babelrc, is it an intentional add?

Reply:
I think we can put it in the webpack.config.js since it's gonna get merged? @JackUrb

Reply:
Leaving this for now.